package.json

```diff
@@ -450,6 +450,16 @@
           "default": false,
           "description": "Run 'go test -coverprofile' on save"
         },
+        "go.showCovered": {
```

Comment:
Am a little hesitant to add new settings. The list of settings we have is really blowing up.
How about re-using the existing setting for coverage.

```
"go.coverOnSave": {
   "type": ["boolean", "string"],
   "enum": [true, "showCoveredOnly", "showUncoveredOnly"],
   "default": true,
   "description": "..."
}
```